### PR TITLE
Feature logger

### PR DESCRIPTION
**Pull request by JorisBelier**
Added a few log calls in several classes. Also deleted a class that already existed a library for.

### CODE REVIEW

File: core/src/main/java/com/sem46/core/Enemy.java [hunk 1]
```diff
@@ -20,6 +20,8 @@ public Enemy(Vector2 start, float size, Direction dir, float speed) {
         else {
             velocity.x = -speed;
         }
+        
+        Logger.log("New enemy : [size:=" + size+",speed:=" + velocity.x+"]" );
```
Comment (ymte) [inline core/src/main/java/com/sem46/core/Enemy.java]: deze regel code is vet

File: core/src/main/java/com/sem46/core/GameScreen.java [hunk 2]
```diff
@@ -31,6 +31,7 @@ public void returnMainMenu(){
 
     public void updateHighscore(int score){
         Settings.addScore(score);
+        Logger.log("Save score: " + score);
         Settings.save();
```
Comment (ymte) [inline core/src/main/java/com/sem46/core/GameScreen.java]: haha nice

File: core/src/main/java/com/sem46/core/Logger.java [hunk 3]
```diff
@@ -41,22 +42,4 @@ public static void log(String s){
...
 	}
 	
-	private static class ListModelLogger extends AbstractListModel{
-		
-		
-		
-		@Override
-		public Object getElementAt(int arg0) {
-			// TODO Auto-generated method stub
-			return list.get(arg0);
```
Comment (ymte) [inline core/src/main/java/com/sem46/core/Logger.java]: i cri evrytiem

**Comment by ymte** [comment]
nice pull request

**Comment by ashay123** [comment]
Nice, merged.

**Pull request merged**